platforms/intellij/skate/README.md

```diff
@@ -16,10 +16,14 @@ We use this at Slack for several use cases:
 2. Add custom plugin repository link from "Manage Plugin Repositories"
 3. Search "Skate" in the plugins marketplace and install it
 
-#### Local testing
+#### Local testing with zip
 1. Build local version of the plugin with `./gradlew buildPlugin`
 2. Open IDE settings, then "Install Plugin from Disk..."
 
+#### Local testing from CLI
```

Comment:
I'd call this "Local testing with `runLocalIde`" since this command can be run from the IDE too